agent-inject/agent/agent_test.go

```diff
@@ -23,6 +24,9 @@ func testPod(annotations map[string]string) *corev1.Pod {
 							MountPath: "serviceaccount/somewhere",
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
+						RunAsUser: &RunAsUser,
+					},
```

Comment:
I don't think we want every test to be run against a Pod with the SecurityContext set like this. It should be set per-test because this is not going to be set in every Pod on every system.

Reply:
Good point!